src/js/api/errors.js

```diff
@@ -78,15 +78,25 @@ export const FLASH_MEDIA_ERROR = 214000;
  * @param {code} [ErrorCode] - The error code.
  * @param {sourceError} [Error] - The lower level error, caught by the player, which resulted in this error.
  */
-export class PlayerError extends Error {
+export class PlayerError {
     constructor(message, code, sourceError = null) {
-        // Avoid passing message via super so that we can define an enumerable message property on PlayerError
-        super();
-        this.code = isValidNumber(code) ? code : null;
+        this.code = isValidNumber(code) ? code : 0;
```

Comment:
@johnBartos Note that this would result in errors with code `0` being pinged back by our analytics plugin. Since we should not use `PlayerError` without specifying a code, I'm OK with this, since the data would indicate exactly that issue.